src/remotestorage.js

```diff
@@ -763,8 +763,12 @@
     },
 
     localStorageAvailable: function () {
```

Comment:
But what about the `nocache` build? There you still want to save config in localStorage, even though that caching feature is not available.